Package@swift-5.9.swift

```diff
@@ -34,11 +37,18 @@ let package = Package(
             plugins: [
                 .plugin(name: SwiftLint.plugin),
             ]),
+        .target(
+            name: "SnapshotTestingExtensions",
+            swiftSettings: .swiftSettings,
+            plugins: [
+                .plugin(name: SwiftLint.plugin),
+            ]),
         .testTarget(
             name: "LayoutTests",
             dependencies: [
                 "Layout",
                 "Nimble",
+                "SnapshotTestingExtensions",
```

Comment:
Place immediately following Layout so it is together with the other target in this package.
```suggestion
                "Layout",
                "SnapshotTestingExtensions",
                "Nimble",
```